config/src/config/api_config.rs

```diff
@@ -83,6 +87,8 @@ impl Default for ApiConfig {
             max_account_resources_page_size: DEFAULT_MAX_ACCOUNT_RESOURCES_PAGE_SIZE,
             max_account_modules_page_size: DEFAULT_MAX_ACCOUNT_MODULES_PAGE_SIZE,
             max_gas_view_function: DEFAULT_MAX_VIEW_GAS,
+            max_runtime_workers: None,
+            runtime_worker_amplifier: 2,
```

Comment:
So, the smallest amount you can increase it is by 50%.  Should we use a float to represent smaller amounts?

Reply:
I think a float is probably overkill. If folks want to be more specific, they could just set `max_runtime_workers` directly.